project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version = 0.13.16
```

Comment:
Newline is missing.

Reply:
added this in all build.properties files in all tools/utils